pbiviz.json

```diff
@@ -2,14 +2,14 @@
   "visual": {
     "name": "drilldownCartogram",
     "displayName": "Drilldown cartogram",
-    "guid": "drilldownCartogramD5E1077C70074FFCB26AFC2A636614BF",
+    "guid": "drilldownCartogramD5E1077C70074FFCB26AFC2A636618BF",
     "visualClassName": "Visual",
     "version": "1.0.0",
```

Comment:
Please increase version in order to submit this visual to Office Store.

Reply:
Done